app/helpers/container_helper/textual_summary.rb

```diff
@@ -9,7 +9,7 @@ def textual_group_properties
   end
 
   def textual_group_relationships
-    items = %w(ems container_group container_project)
+    items = %w(ems container_node container_group container_project)
```

Comment:
@zakiva actually it should be:

    items = %w(ems container_project container_group container_node )